curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -58,6 +58,16 @@ static ClientOpMetric* g_clientOpMetric = nullptr;
 
 DECLARE_int32(v);
 
+DEFINE_int32(vlog_level, 0, "set v log level");
+static bool CheckVLogLevel(const char*, int32_t value) {
+    if (value >= 0 && value <= 9) {
+        FLAGS_v = value;
+        return true;
+    }
+    return false;
+}
+DEFINE_validator(vlog_level, CheckVLogLevel);
```

Comment:
Add comments to explain the logic of the modification.

Reply:
> Add comments to explain the logic of the modification.

fix